README.md

```diff
@@ -1,3 +1,86 @@
-# Developer Rubric Discord Bot
+<div id="top"></div>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://www.semasoftware.com/">
+    <img src="https://cdn.discordapp.com/icons/972616580078907442/3660c78f24a43ee25aa2e1a313d18424.webp?size=160" alt="Logo" width="80" height="80" >
+  </a>
+
+  <h3 align="center">Developer Rubric Discord Bot</h3>
+
+  <p align="center">
+    <a href="">View Demo</a>
```

Comment:
Could we comment this part out, until there is a demo?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question